src/content/dvt/index.md

```diff
@@ -0,0 +1,106 @@
+---
+title: Distributed Validator Technology
+description: Distributed Validator Technology enables the distributed operation of an Ethereum validator by multiple parties.
+lang: en
+---
+
+# Distributed Validator Technology {#what-is-dvt}
```

Comment:
```suggestion
```
we typically do not include this H1 header in addition to the title defined in the frontmatter. It also had the same page ID as the following section.